app/views/PlatformInitiativesListPage.scala.html

```diff
@@ -0,0 +1,137 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.mvc.Call
+@import uk.gov.hmrc.cataloguefrontend.DateHelper._
+@import uk.gov.hmrc.cataloguefrontend.connector.PlatformInitiative
+@import uk.gov.hmrc.cataloguefrontend.connector.PlatformInitiativesConnector
+@import uk.gov.hmrc.cataloguefrontend.service.PlatformInitiativesService
+@import uk.gov.hmrc.cataloguefrontend.ViewMessages
+@import helper._
+@import scala.concurrent.Future
+@import java.net.URL
+@import scala.util.Try
+
+@this(viewMessages: ViewMessages, platformInitiativesService: PlatformInitiativesService)
+
+@(initiatives          : Seq[PlatformInitiative],
+displayChart           : Boolean,
+displayProgress        : Boolean,
+form                   : Form[_]
+)(implicit messages    : Messages, request: Request[_])
+
+@standard_layout(s"Platform Initiatives", "platforminitiatives") {
+<header>
+    <h1>Platform Initiatives</h1>
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+</header>
+
+@if(form.hasErrors) {
+<div class="alert alert-danger" role="alert">
+    <ul class="list">
+        @form.errors.map { error =>
+        <li class="alert-danger"> @Messages(error.message, error.args: _*)</li>
+        }
+    </ul>
+</div>
+}
+
+<div id="platform-initiatives">
+    <form action="/platform-initiatives" method="get">
+        <div class="col-xs-1 padding-reset-right">
+            <label for="search">Search:</label>
+        </div>
+        <div class="col-xs-5">
+            <input class="search form-control" id="search" type="text" name="initiativeName"
+                   value='@form("initiativeName").value'>
+        </div>
+        <div class="right" style="line-height:2.2em;">
+            <a href="?chart=true&progress=false">Pie chart</a> |
+            <a href="?chart=false&progress=true">Progress bar</a>
+        </div>
+        <br><br><br>
+        <table class="table table-striped">
+            <tbody class="list">
+            @initiatives.zipWithIndex.map{case (initiative, i) =>
+                @if(initiative.targetProgress != 0){
+                <tr class="col-sm-6">
+                    <td>
+                        @if(displayChart && !displayProgress){
```

Comment:
would this read better as
```
<div class="initiativeName" style="width:500px;">
  <h3>@initiative.initiativeName</h3>
  if (displayChart) { ...} else {... }
  if (displayProgress) { ...} else {... }
</div>
```
? (I could be mistaken)

Reply:
The reason I've done it the way I have is I want them to be independent. I.E I want to check if just one is True or if both are True, so there are 3 conditions - Chart = true, Progress = true, Chart && Progress = true. I'll see if I can make it slightly more readable, though!